test/types/imag/opequals2.2.good

```diff
@@ -5,4 +5,4 @@ $CHPL_HOME/modules/internal/ChapelBase.chpl:nnnn: note: is passed to formal 'ref
 opequals2.chpl:6: note: other candidates are:
 $CHPL_HOME/modules/internal/ChapelBase.chpl:nnnn: note:   -=(ref lhs: int(8), rhs: int(8))
 $CHPL_HOME/modules/internal/ChapelBase.chpl:nnnn: note:   -=(ref lhs: int(16), rhs: int(16))
-note: and 22 other candidates, use --print-all-candidates to see them
+note: and 21 other candidates, use --print-all-candidates to see them
```

Comment:
I could have sworn the prediff handled number of candidates as well, but I might be thinking of a different prediff ...

Reply:
We do have some special case prediffs that handle the number of candidates, but not here. And no system wide one (which this test and others aren't using anyways)